energy.md

```diff
@@ -414,7 +419,26 @@ Note that the size classes shown are tailored to each region, depending on the d
 
 ## Energy Trade
 
-GCAM models trade for coal, gas, oil, and bioenergy using a Heckscher-Ohlin paradigm. That is, trade for these products occurs freely into and out of global markets. Other energy carriers (e.g., solar, wind, geothermal) are not traded. For more information, see the [discussion of approaches to international trade](trade.html).
+GCAM models trade for coal, gas, oil, and bioenergy using an Armington approach that is described below. Under this approach regions are allowed to choose between domestically produced products or globally traded products when making a consumption decision. This approach allows for the computation of a regionally distinct consumption price for fossil fuels based on the domestic and imported consumption. 
+
+Other energy carriers (e.g., solar, wind, geothermal) are not traded. For more generalinformation, see the [discussion of approaches to international trade](trade.html).
+
+### Fossil fuel trade in GCAM
+
+ The figure below depicts the fossil fuel trade structures (using coal as an example). In previous versions of GCAM, every region produced and consumed from a single global market. All crude oil, coal, and natural gas production was sent to a shared market, from which, every region consumed. Only net trade could be tracked and supply was affected by global rather than regional demand. The current structure maintains a global market (e.g. traded coal), but distinguishes between direct consumption of domestic resources and consumption of imported fossil fuels.
+ 
+ <img src="gcam-figs/Fossil_fuel_trade.png" width="900" height="400" /><br/>
+** Schematics of the structures for the flows of the "Coal" commodity in GCAM, with only 3 regions shown for simplicity.**
```

Comment:
Didn't render correctly.  You need to get rid of the space between `**` and `Schematics`

Not sure if this was necessary as well but maybe also get rid of the leading space in front of `<img`

Reply:
I took the spaces out. Thanks for the catch.